crates/bitwarden/src/vault/client_vault.rs

```diff
@@ -1,8 +1,12 @@
-use super::sync::{sync, SyncRequest, SyncResponse};
-use crate::{error::Result, Client};
+use super::{
+    repository::CipherSqliteRepository,
+    sync::{sync, SyncRequest, SyncResponse},
+};
+use crate::{error::Result, vault::cipher::repository::CipherRepository, Client};
 
 pub struct ClientVault<'a> {
     pub(crate) client: &'a mut crate::Client,
+    pub cipher_repository: Box<dyn CipherRepository + Send>,
```

Comment:
I think repositories should be internal to the module and we probably want to expose some service instead. Since we will want to hook into some event system, and probably expose nice methods for saving decrypted views.